ruby-3.0.2-circleci/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ruby:3.0.2-buster
+MAINTAINER ParkWhiz <dev@parkwhiz.com>
+
+WORKDIR /
+
+
+# -----------------------
+# -- Installing packages.
+# -----------------------
+
+RUN \
+  apt-get update && \
+  apt-get install -y --no-install-recommends nano vim postgresql
+
+
+# ---------------------
+# -- Installing Docker.
```

Comment:
circle is running docker inside docker?

Reply:
<img width="803" alt="Screen Shot 2021-08-13 at 11 41 39 AM" src="https://user-images.githubusercontent.com/802806/129392755-7fe4df65-1523-4f0f-a2f3-34937abda5c5.png">
🙂

Reply:
yeah, needs it for `docker build`. i'd have thought that the `setup_remote_docker` would set that up, but i'd have been incorrect.